bash/ubuntu-run.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# If set to true, ddtrace-run will be enabled for dbt-server.
+readonly dbt_server_enable_ddtrace="${DBT_SERVER_ENABLE_DDTRACE-false}"
+# How many dbt server worker processes.
+readonly dbt_server_worker="${DBT_SERVER_WORKER_NUM-3}"
+# Dbt server listening port.
+readonly dbt_server_port="${DBT_SERVER_WORKER_PORT-8585}"
+# Max # of requests that dbt server is allowed.
+readonly dbt_server_max_requests="${DBT_SERVER_MAX_REQUESTS-5}"
+
+gunicorn="gunicorn"
+if [ "${dbt_server_enable_ddtrace}" = "true" ]; then
+    dd_trace="ddtrace-run gunicorn"
+fi
+
+sh bash/start-celery.sh
+sh bash/ubuntu-start-redis.sh
+
+${gunicorn} dbt_server.server:app --workers ${dbt_server_worker} \
```

Comment:
I understand not being able to start multiple parent processes in docker-- does this change mean that we wouldn't include any args [here](https://github.com/dbt-labs/runtime-gateway/blob/main/pkg/provider/kubernetes/provider.go#L346) when we create server containers via the runtime? 

If so, we need to think about how to manage this logic difference in the `runtime-gateway` -- we may need to just add a script to server 0.1.0 to start gunicorn alone, so that we don't have to think about version differences in `runtime-gateway`.

Reply:
The logic here is exactly I copied from runtime gateway, the default param value is copied as well.

In an ideal world, this bash script should takes bunch of env vars, configuring gunicorn and that's it. User(either IDE or runtime gateway should only set env var but not touch execution command). WDYT?

Reply:
I think that makes sense! We'll just want to make sure we backport that to server 0.1.0 so that we don't have to have conditional logic in the `runtime-gateway`.

Reply:
Definitely fine to revisit-- runtime gateway/semantic layer will continue to use 0.1.0 until dbt 1.4 is fully deprecated-- server 0.2.0 will never be compatible with those earlier versions